src/app/posts/posts.module.ts

```diff
@@ -17,6 +17,7 @@ import {RouterModule} from '@angular/router';
 import {HeaderComponent} from 'src/app/header/header.component';
 import {PostDetailsComponent} from 'src/app/post-details/post-details.component';
 import { PostRoutingModule } from './post-routing.module';
+import { CommentModule } from '../post-details/comments/comment.module';
```

Comment:
feature modulokat ne ágyazzunk be másik feature modulba, ki kellene tenni a comments modul mappát egy szintre a mappa struktúrába